api/models.py

```diff
@@ -1,11 +1,39 @@
 from django.db import models
+from . import spellcheck
 
 # Create your models here.
-class Post(models.Model):
+class Document(models.Model):
     title = models.CharField(max_length=50)
-    content = models.TextField()
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
-        return self.title
\ No newline at end of file
+        return self.title
+
+class Block(models.Model):
+    block_document = models.ForeignKey(Document, on_delete=models.CASCADE)
+    """
+    The parent document of a block.
+    """
+    
+    block_content = models.TextField()
+    """
+    The text content of a block.
+    """
+    
+    block_order = models.IntegerField(default=0, unique=True)
+    """
+    The index of a block within a document.
+    """
+    
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.block_content[:10]
```

Comment:
Q: unsure as to why this only returns content unto 10?

Reply:
Yeah it should only return the block content.